scripts/xjoin-config/my-pipeline.json

```diff
@@ -0,0 +1,34 @@
+{
+    "description": "Ingest pipeline for xjoin.inventory",
+    "processors": [
```

Comment:
Is there reference or motivation why we use only three processors? and for these specific fields only?

Reply:
I not sure about why only these three processors, It is just the inventory team config for indexing there data in elasticsearch which is copied here.